src/shared/pages/dashboard.tsx

```diff
@@ -39,6 +39,9 @@ const Dashboard: React.FC = () => (
                             <li>
                                 <Link className="govuk-link" to="/user-search">Manage a user</Link>
                             </li>
+                            <li>
+                                <Link className="govuk-link" to="/add-user">Add a user</Link>
```

Comment:
```suggestion
                                <Link className="govuk-link" to="/add-user">Add a new user</Link>
```

Reply:
This is following the existing labelling pattern in MUI.